src/common.c

```diff
@@ -78,14 +78,30 @@ int fi_poll_fd(int fd, int timeout)
 	return ret == SOCKET_ERROR ? -ofi_sockerr() : ret;
 }
 
+/*
+ * Find the location of the most significant bit set. Value 1~64
+ * means such bit is found. Value 0 means no bit is set.
+ */
+static inline int msb(uint64_t x)
+{
+	int pos = 0;
+
+	while (x) {
+		x >>= 1;
+		pos++;
+	}
+
+	return pos;
+}
```

Comment:
There's an existing function, fi_size_bits(), that I think does the same thing.  Maybe we can rename fi_size_bits() to ofi_msb() as a common function.

Reply:
Yes you are right, fi_size_bits() does the same thing although it's not obvious from the function name. Renaming the function sounds good. What is the guideline to use "ofi_" prefix vs "fi_" prefix for non-API functions?

Reply:
The ofi prefix is for all things internal.  The fi prefix should be reserved for public facing APIs and structures.  There's just a lot of fi prefix still in use that has never been converted to ofi.  I may try to work on that this week.  It's just a lot of potential changes.

Reply:
fi_size_bits() behaves slightly differently. When the input is 0, fi_size_bit() would return 1 which is incorrect. msb() returns 0 in this case.